app/src/main/java/com/cornellappdev/transit/ui/components/MenuItem.kt

```diff
@@ -24,16 +28,17 @@ import androidx.compose.ui.unit.sp
  * @param sublabel The sublabel for the item
  */
 @Composable
-fun MenuItem(icon: ImageVector, label: String, sublabel: String) {
+fun MenuItem(icon: ImageVector, label: String, sublabel: String, onClick: () -> Unit) {
```

Comment:
Sorry for not commenting this last time, but let's make this `onClick` actually get called!

You can place a `.clickable()` modifier on the `Row` to make the row respond to taps. Make the `onClick` function passed here get called when the Row is tapped on!

Reply:
done! :)